tests/input/examples/personinfo_basic/transform/personinfo-to-agent.transform.yaml

```diff
@@ -25,13 +25,39 @@ class_derivations:
         populated_from: name
 
       age:
-        expr: "str({age_in_years})+' years'"
+        expr: "str({age_in_years}) + ' years'"
 
+      # YAML interprets this as expr = None
+      # no transformation is performed
       primary_email:
+        expr: NULL
 
+      # this is the string "NULL", which will set this field to null
+      secondary_email:
+        expr: "NULL"
+
+      # FIXME: this should return the type "None", rather than null
```

Comment:
This is dodgy -- not sure what the intention is, whether "None" should be treated as a string by the transformer or whether it should be treated as null (the current MO). IMHO it should be a string as "NULL" already sets the field to null in the transformation.